File: src/Genesis/Editor/Editor.cpp
#include <Genesis/Client/Client.hpp>
#include <Genesis/Server/Server.hpp>
#include <Runtime/Common/Common.hpp>
#include <iostream>

int main(int argc, char** argv) {
	std::cout << "Starting Editor..." << std::endl;

	ge::server::Server::init();
	ge::client::Client::init();
	rt::common::Common::init();

	std::cin.get();
}

File: src/Genesis/Editor.lua
project "Genesis.Editor"
	kind "ConsoleApp"
	location "%{wks.location}/src/Genesis/Editor/"

	files {
		"%{prj.location}/**.c",
		"%{prj.location}/**.h",
		"%{prj.location}/**.cpp",
		"%{prj.location}/**.hpp"
	}

	includedirs {
		includeDirs["Genesis"],
		includeDirs["spdlog"],
		includeDirs["fmt"]
	}

	links {
		"ThirdParty.spdlog",
		"ThirdParty.fmt",
		"Runtime.Common",
		"Genesis.Core",
		"Genesis.Server",
		"Genesis.Client"
	}
